validation/standard/standard-versions-sepolia.toml

```diff
@@ -0,0 +1,25 @@
+standard_release = "op-contracts/v1.6.0"
+
+[releases]
+
+# Contracts which are
+#   * unproxied singletons: specify a standard "address"
+#   * proxied             : specify a standard "implementation_address"
+#   * neither             : specify neither a standard "address" nor "implementation_address"
+
+# Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.6.0
+[releases."op-contracts/v1.6.0"]
```

Comment:
TODO: need to get sepolia v1.6.0 contract addresses.

Reply:
Here are the addresses I have, please do verify these though:

```
optimism_portal: 0x35028bae87d71cbc192d545d38f960ba30b4b233
system_config: 0xCcdd86d581e40fb5a1C77582247BC493b6c8B169
delayed_weth: 0x07f69b19532476c6cd03056d6bc3f1b110ab7538
dispute_game_factory: 0xa51bea7e4d34206c0bcb04a776292f2f19f0beec
mips: 0x47B0E34C1054009e696BaBAAd56165e1e994144d
preimage_oracle: 0x92240135b46fc1142dA181f550aE8f595B858854
l1_cross_domain_messenger: 0xD3494713A5cfaD3F5359379DfA074E2Ac8C6Fd65
l1_erc721_bridge: 0xae2af01232a6c4a4d3012c5ec5b1b35059caf10d
l1_standard_bridge: 0x64b5a5ed26dcb17370ff4d33a8d503f0fbd06cff
optimism_mintable_erc20_factory: 0xe01efbeb1089d1d1db9c6c8b135c934c0734c846
```

Reply:
```
optimism_portal: 0x35028bae87d71cbc192d545d38f960ba30b4b233
```
matches against `op-contracts/v1.4.0-rc.3` tag

---
```
system_config: 0xCcdd86d581e40fb5a1C77582247BC493b6c8B169
```
matches against `op-contracts/v1.4.0-rc.3` tag

---
```
delayed_weth: 0x07f69b19532476c6cd03056d6bc3f1b110ab7538
```
matches against `op-contracts/v1.6.0` tag

---
```
dispute_game_factory: 0xa51bea7e4d34206c0bcb04a776292f2f19f0beec
```
matches against `op-contracts/v1.6.0` tag

---
```
mips: 0x47B0E34C1054009e696BaBAAd56165e1e994144d
```
matches against `op-contracts/v1.6.0` tag

---
```
preimage_oracle: 0x92240135b46fc1142dA181f550aE8f595B858854
```
matches against `op-contracts/v1.6.0` tag

---
```
l1_cross_domain_messenger: 0xD3494713A5cfaD3F5359379DfA074E2Ac8C6Fd65
```
matches against `op-contracts/v1.4.0-rc.3` tag

---
```
l1_erc721_bridge: 0xae2af01232a6c4a4d3012c5ec5b1b35059caf10d
```
matches against `op-contracts/v1.4.0-rc.3` tag

---
```
l1_standard_bridge: 0x64b5a5ed26dcb17370ff4d33a8d503f0fbd06cff
```
matches against `op-contracts/v1.4.0-rc.3` tag

---
```
optimism_mintable_erc20_factory: 0xe01efbeb1089d1d1db9c6c8b135c934c0734c846
```
matches against `op-contracts/v1.4.0-rc.3` tag